app/models/exercise.rb

```diff
@@ -36,9 +38,14 @@ class Exercise < ApplicationRecord
   validates_presence_of :submissions_count,
                         :guide, :bibliotheca_id
 
+  alias_attribute :parent, :guide
```

Comment:
We are removing the concept of both parent and children in favor of structural_* and navigable_*.
In this case, we should use structural_parent which is already defined.

Reply:
So I should delete this alias (and the other one) and just use structural_parent/structural_children in flow?